plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/util/DeltaLakeWriteUtils.java

```diff
@@ -52,11 +59,14 @@
 import static io.trino.spi.type.TinyintType.TINYINT;
 import static java.lang.Math.floorDiv;
 import static java.lang.Math.floorMod;
+import static java.lang.String.format;
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 // Copied from io.trino.plugin.hive.util.HiveWriteUtils
 public final class DeltaLakeWriteUtils
 {
+    public static final String PARQUET_BLOOM_FILTER_COLUMN_ENABLED_PREFIX = "write.parquet.bloom-filter-enabled.column.";
```

Comment:
@raunaqmorarka / @ebyhr how comfortable would you be adding support for a table property like this one?

I don't see any serious side-effects here. If Delta Lake will introduce support for Bloom Filters through https://github.com/delta-io/delta/issues/2751 we will simply mark obsolete the property and use the new property.

@jkylling consider adding "trino" in the prefix name - this is nothing "official" in delta.

Reply:
I don't see a problem with adding it, but I would like to at least see if there is any initial opinion about it from delta spec.